src/palindrom-event-target.js

```diff
@@ -0,0 +1,33 @@
+/*! Palindrom
+ * https://github.com/Palindrom/Palindrom
+ * (c) 2019 Omar Alsahker
+ * MIT license
+ */
+
+// privatize the prop
+const listeners = Symbol('listeners');
+
+export class PalindromEventTarget {
```

Comment:
Was the removal of the callbacks signed off? I would not agree to this, at least without proper discussion, because changing callbacks to events is something that was already tried in the past and then removed (actually by you):

- Feb 2015, some events were added: https://github.com/Palindrom/Palindrom/commit/05e5bfee9f16a14dfd0cbec56a6e339dbe56de82
- Oct 2015, discarded proposal to change more callbacks to events: https://github.com/Palindrom/Palindrom/issues/74
- Feb 2016, discarded propososal to remove events and use callbacks: https://github.com/Palindrom/Palindrom/issues/78
- Jun 2016, agreed proposal to remove remove events and use callbacks: https://github.com/Palindrom/Palindrom/issues/74#issuecomment-234239589
- Apr 2017, events removed by @alshakero in commit https://github.com/Palindrom/Palindrom/commit/c5f95a090fcce5f6a2a9981e83f8bd0b8f1ca28b#diff-92b698a38d3db0381546f440a80e631b, part of https://github.com/Palindrom/Palindrom/pull/130